packages/google_maps_flutter/google_maps_flutter_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
```

Comment:
This will need a version bump.

Reply:
We will need a version bump before this lands. I was thinking it would make sense to do so when we are closer to landing due to this being a federated change that must be split into multiple steps.

Reply:
> due to this being a federated change that must be split into multiple steps.

I don't follow. Are there other changes that aren't uploaded yet? I would not expect this to need changes to other packages.

Reply:
I confused this with a different and rather similar PR. Never mind that.